src/bp-blogs/bp-blogs-cache.php

```diff
@@ -27,7 +27,7 @@ function bp_blogs_update_meta_cache( $blog_ids = false ) {
 		'object_ids'    => $blog_ids,
 		'object_type'   => buddypress()->blogs->id,
 		'object_column' => 'blog_id',
-		'cache_group'   => 'blog_meta',
+		'cache_group'   => 'bp_blog_meta',
```

Comment:
Please update `bp_blog` to `bb_blog`. We should have`bb` prefix.

Reply:
@dipskakadiya updated the PR as per request.